Nasal/engine.nas

```diff
@@ -128,24 +125,10 @@ var oil_consumption = maketimer(1.0, func {
             consumption_qph = consumption_qph * 1.3;
         }
 
-        # Consumption also raises with oil in service time (lower viscosity => more friction)
-        # (Oil should be changed at 50 hrs!)
-        # See: http://www.t-craft.org/Reference/Aircraft.Oil.Usage.pdf
-        # Hours:        0 |    10 |    25 |  50   |    75
-        # Add Qts/hr:   0 |  0.02 | 0.125 | 0.5   | 1.125
```

Comment:
You could probably leave these comments in the code in case it gets implemented in the future.